docs/reference/modules/network/tracers.asciidoc

```diff
@@ -36,6 +36,42 @@ PUT _cluster/settings
 }
 --------------------------------------------------
 
+By default, the tracer logs a summary of each request and response which
```

Comment:
I'm sure you already thought about it, but this is concerning if applied to REST APIs like Get API Key. We should maybe exclude sensitive APIs?

Reply:
It's a good question. I don't see that this feature differs much from the long-standing `ESLoggingHandler` which can be set to log the raw contents of every transport message. `TRACE` logs can be rather sensitive. But I would like other opinions on this.

Reply:
^ particularly Tim's opinion (or at least Tim's opinion about whose opinion I should seek)

Reply:
We have some concept of filtering rest request (but not response) bodies today (*), however it's specific to security and only really used for audit logging.

I'm torn. It's only TRACE, it's off by default and even when it's on most of our log shipper configurations will drop TRACE messages, but it's one of those things that someone could raise as an issue later on. We probably need to get wider agreement first.

@jkakavas can we have a chat about this sometime?

(*) https://github.com/elastic/elasticsearch/blob/e059dd08f39b2197f4e81bb5368e869f71992dd6/server/src/main/java/org/elasticsearch/rest/RestRequestFilter.java#L33